.github/workflows/maven-macos.yml

```diff
@@ -26,10 +26,15 @@ jobs:
       run: mkdir staging
     - name: Build JAR with Maven
       run: mvn -B package
-    - name: Build and package as MacOS application with Gluon
-      run: mvn client:build client:package
-      env:
-        GRAALVM_HOME: /System/Volumes/Data/Users/runner/hostedtoolcache/GraalVM/20.2.0-java11/x64/
+    - name: Gluon Build
+      uses: gluonhq/gluon-build@v1
+      with:
+        gluon-license: ${{ secrets.GLUON_LICENSE }}
+        target: 'macos'
```

Comment:
use 'desktop' here. that will build for the 'host' client target, which is macos